### PR TITLE
Fix #4927 - Add rabin2 -qqz to list only strings (no offsets or sizes)

### LINKED ISSUE #4927: Add rabin2 -qqz to list only strings (no offsets or sizes)

### PR DESCRIPTION
**Pull request by nevun**
$ rabin2 -qqz /bin/dd |head -3 
out of memory
unknown operand %s
no value specified for %s

$ rabin2 -qz /bin/dd |head -3  
0x1163e0 14 13 out of memory
0x1163ee 19 18 unknown operand %s
0x116401 26 25 no value specified for %s

$ rabin2 -z /bin/dd |head -3  
vaddr=0x001163e0 paddr=0x000163e0 ordinal=000 sz=14 len=13 section=.rodata type=ascii string=out of memory
vaddr=0x001163ee paddr=0x000163ee ordinal=001 sz=19 len=18 section=.rodata type=ascii string=unknown operand %s
vaddr=0x00116401 paddr=0x00016401 ordinal=002 sz=26 len=25 section=.rodata type=ascii string=no value specified for %s

### CODE REVIEW

File: binr/rabin2/rabin2.c [hunk 1]
```diff
@@ -532,7 +533,12 @@ int main(int argc, char **argv) {
 			set_action (R_BIN_REQ_VERSIONINFO);
 			break;
 		case 'V': set_action (R_BIN_REQ_VERSIONINFO); break;
-		case 'q': rad = R_CORE_BIN_SIMPLE; break;
+		case 'q':
+			if (rad & R_CORE_BIN_SIMPLE)
+				rad = R_CORE_BIN_SIMPLEST;
```
Comment (radare) [inline binr/rabin2/rabin2.c]: use braces or ternary operator

**Comment by radare** [comment]
just minor style fix. can you add a test case for this?

do this thing works with `izqq` ?

**Comment by radare** [comment]
ref https://github.com/radare/radare2/issues/4927

**Comment by nevun** [comment]
> do this thing works with izqq ?

no.. libr/core/cmd_info.c looks a bit more hairy to fix though.. need to add test case before touching it I think.. :)

**Comment by alvarofe** [comment]
i'm doing `izqq`. I will merge and create test. Thx

**Comment by alvarofe** [comment]
merged and test

**Pull request closed without merging**